src/Renderer/c3DEngine.js

```diff
@@ -32,6 +32,7 @@ function c3DEngine(scene, positionCamera, viewerDiv, debugMode, gLDebug) {
     this.scene3D.sortObjects = false;
     this.width = this.debug ? viewerDiv.clientWidth * 0.5 : viewerDiv.clientWidth;
     this.height = viewerDiv.clientHeight;
+    this.viewerDiv.size = new THREE.Vector2(this.width, this.height);
```

Comment:
Don't modify `viewerDiv` please (this isn't what my original comment suggested).